src/nvim/message.c

```diff
@@ -606,6 +713,19 @@ static bool emsgfv(const char *fmt, va_list ap)
   return emsg((const char_u *)errbuf);
 }
 
+static bool emsgfv_echo(const char *fmt, va_list ap)
```

Comment:
This function is short and not used anywhere, but in emsgf_echo, so should not exist.